repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc

```diff
@@ -0,0 +1,133 @@
+= OpenTracing Service
+
+The OpenTracing service (http://opentracing.io/) is a service in *Repose* which allows for sending tracing data to a Tracing service.
+
+The initial release of the OpenTracing service only supports using the Jaeger (http://jaeger.readthedocs.io/en/latest/) Tracer.
```

Comment:
Perhaps:

> ... using the http://jaeger.readthedocs.io/en/latest/[Jaeger client library].

After all, the `Tracer` is just one component in that library, even if it is our entrypoint into pretty much everything else.

Then for the future line, you might say other libraries or other OpenTracing implementations.

Reply:
Updated.